message_app/db/db.py

```diff
@@ -0,0 +1,56 @@
+import sqlite3
+import os
+from flask import current_app, g
+
+# Connect to the database
+# The g name stands for “global”, but that is referring to the data being global within a context.
+# The data on g is lost after the context ends. Flask provides the g object for this purpose.
+# g is a simple namespace object that has the same lifetime as an application context.
+# db is an attribute of object g. We will store the connection to our database to g.db
+# The sqlite3.Row is used to convert the row data in tuple to dictionary for easier access
+def get_db():
+    if 'db' not in g:
+        g.db = sqlite3.connect(
+            current_app.config['DATABASE'],
+            detect_types=sqlite3.PARSE_DECLTYPES
+        )
+        g.db.row_factory = sqlite3.Row
+
+    return g.db
+
+
+# Close the database connection
+# We close the connection to our database by removing it from the g object
```

Comment:
Wrong wording. We close the connection by calling` db.close()`, not by removing it from `g`. You can say `Remove the database connection from g and close it`.